component-definition.json

```diff
@@ -144,8 +144,23 @@
               "page": {
                 "resourceType": "core/franklin/components/block/v1/block",
                 "template": {
-                  "name": "data-cards",
-                  "model": "data-cards"
+                  "name": "Data Cards",
+                  "filter": "data-cards"
```

Comment:
Raise these component json files in a separate PR so that we can at least merge and test the authoring part.